### PR TITLE
Merge bootblock versions for fw6

### PR DESCRIPTION
**Pull request by IgorBagnucki**
Signed-off-by: Igor Bagnucki <igor.bagnucki@3mdeb.com>

### CODE REVIEW

File: src/mainboard/protectli/vault_kbl/bootblock.c [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#include <bootblock_common.h>
+#include <device/pnp_ops.h>
+#include <superio/ite/common/ite.h>
+#include <superio/ite/it8613e/it8613e.h>
+
+#define GPIO_DEV PNP_DEV(0x2e, IT8613E_GPIO)
+#define UART_DEV PNP_DEV(0x2e, IT8613E_SP1)
+
+/* Return an id of the installed chip. */
+static int ite_read_id(void)
```
Comment (miczyg1) [inline src/mainboard/protectli/vault_kbl/bootblock.c:12]:
```suggestion
static uint16_t ite_read_id(void)
```

File: src/mainboard/protectli/vault_kbl/bootblock.c [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+{
+	pnp_enter_conf_state(GPIO_DEV);
+	int id = (pnp_read_config(GPIO_DEV, 0x20) << 8)
```
Comment (miczyg1) [inline src/mainboard/protectli/vault_kbl/bootblock.c:15]:
```suggestion
	uint16_t id = (pnp_read_config(GPIO_DEV, 0x20) << 8)
```

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: src/mainboard/protectli/vault_kbl/bootblock.c (hunk 1 above)
Comment (IgorBagnucki) [reply src/mainboard/protectli/vault_kbl/bootblock.c]: Fixed

File: src/mainboard/protectli/vault_kbl/bootblock.c (hunk 2 above)
Comment (IgorBagnucki) [reply src/mainboard/protectli/vault_kbl/bootblock.c]: Fixed

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**